coordinator/src/services/translation/request/dispense/dispense-notification.ts

```diff
@@ -277,16 +295,10 @@ export function getOrganisationPerformer(fhirFirstMedicationDispense: fhir.Medic
   return fhirFirstMedicationDispense.performer.find(p => p.actor.type === "Organization")
 }
 
-export function getFhirGroupIdentifierExtension(
+export function getMedicationDispenseContained(
```

Comment:
is this still used? i think it would be neater to use this in all places we reference `contained[0]`

Reply:
changed it in most places in coordinator